ibis_bigquery/compiler.py

```diff
@@ -7,7 +7,7 @@
 import ibis
 
 try:
-    import ibis.backends.base_sqlalchemy.compiler as comp
```

Comment:
Based on the test failures, this is the name in 1.4.0.

Let's add another try-except block and fall back to that.